tailwind.config.js

```diff
@@ -2,7 +2,21 @@ module.exports = {
   purge: ['./src/pages/**/*.ts', './src/pages/**/*.tsx', './src/components/**/*.ts', './src/components/**/*.tsx'],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    screens: {
+      sm: '375px',
+    },
+    colors: {
+      main: '#174DE5',
+      green: '#5CB940',
+      red: '#E51718',
+      white: '#FFF',
+      grey: {
```

Comment:
it supports only `gray`

Reply:
In the official document, there's no grey information. :(
I think it will be safer if we use as `gray`.